packages/block-library/src/query-pagination-previous/index.php

```diff
@@ -25,23 +27,39 @@ function render_block_core_query_pagination_previous( $attributes, $content, $bl
 	if ( $pagination_arrow ) {
 		$label = $pagination_arrow . $label;
 	}
-	$content = '';
 	// Check if the pagination is for Query that inherits the global context
 	// and handle appropriately.
 	if ( isset( $block->context['query']['inherit'] ) && $block->context['query']['inherit'] ) {
+		global $wp_query;
+		$max_page               = ! $max_page || $max_page > $wp_query->max_num_pages ? $wp_query->max_num_pages : $max_page;
 		$filter_link_attributes = function() use ( $wrapper_attributes ) {
 			return $wrapper_attributes;
 		};
+
 		add_filter( 'previous_posts_link_attributes', $filter_link_attributes );
-		$content = get_previous_posts_link( $label );
+		if ( 1 !== $paged ) {
+			$content = get_previous_posts_link( $label );
+		} else {
+			// Prints the link only if there are pages to paginate.
+			$content = $max_page > 1 ? sprintf(
+				'<span %1$s>%2$s</span>',
+				$wrapper_attributes,
+				$label
+			) : '';
+		}
 		remove_filter( 'previous_posts_link_attributes', $filter_link_attributes );
-	} elseif ( 1 !== $page ) {
-		$content = sprintf(
+	} else {
+		$content = 1 !== $page ? sprintf(
```

Comment:
I think this goes against the brace style in the PHP guidelines: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/

Reply:
fixed :)